staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -778,6 +778,12 @@ type APIResource struct {
 	SingularName string `json:"singularName" protobuf:"bytes,6,opt,name=singularName"`
 	// namespaced indicates if a resource is namespaced or not.
 	Namespaced bool `json:"namespaced" protobuf:"varint,2,opt,name=namespaced"`
+	// group is the preferred group of the resource.  Empty implies the group of the containing resource list.
```

Comment:
Could you remove the extra space after the period? There are many occurrences. Is it an editor issue?

Reply:
> Could you remove the extra space after the period? There are many occurrences. Is it an editor issue?

MLA when I was growing up I'm afraid.

Reply:
One mystery solved :-)